backend/routes/profile.js

```
@@ -69,13 +77,17 @@ router.post(
 
 // Delete profile photo (reset to default)
-router.delete("/photo-delete",isAuthenticated, async (req, res) => {
+router.delete("/photo-delete", isAuthenticated, async (req, res) => {
   try {
     const { ID_No } = req.query; // Get ID_No from frontend for DELETE
-    if (!ID_No) { return res.status(400).json({ error: "ID_No is required" }); }
+    if (!ID_No) {
+      return res.status(400).json({ error: "ID_No is required" });
+    }
 
     const user = await user.findOne({ user_id: ID_No });
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)